recipes/linkedin/runner.sh

```diff
@@ -9,7 +9,7 @@ VERSION=$DATE
     mkdir -p output
     mkdir -p input
 
-    latest_file=$(axway_ls -nrt LinkedIn | grep .xlsx | tail -1 | awk '{print $NF}')
+    latest_file=$(axway_ls -nrt LinkedIn && grep .xlsx | tail -1 | awk '{print $NF}') || echo 'axway_ls failed on LinkedIn'
```

Comment:
@SteveScott in this file you are changing | to &&. But in met/runner.sh, it's the other way around. Why is that?